src/Symfony/Component/Console/Tests/Output/ConsoleOutputTest.php

```diff
@@ -22,4 +23,19 @@ public function testConstructor()
         $this->assertEquals(Output::VERBOSITY_QUIET, $output->getVerbosity(), '__construct() takes the verbosity as its first argument');
         $this->assertSame($output->getFormatter(), $output->getErrorOutput()->getFormatter(), '__construct() takes a formatter or null as the third argument');
     }
+
+    public function testSetFormatter()
+    {
+        $output = new ConsoleOutput();
+        $outputFormatter = new OutputFormatter();
+        $output->setFormatter($outputFormatter);
+        $this->assertSame($outputFormatter, $output->getFormatter());
+    }
+
+    public function testSetVerbosity()
+    {
+        $output = new ConsoleOutput();
+        $output->setVerbosity(1);
+        $this->assertSame(1, $output->getVerbosity());
```

Comment:
In this particular case you could see it like an Enum, but php doesn't support Enums sadly.

You could probably compare it to:
```php
public function setVerbosity(OutputVerbosityEnum $level);
```

When working with enum/constants, you should forget about the value and think in constant references only to prevent bugs in future versions.